specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -444,24 +428,29 @@ export function Status({
     };
   }, [mergingId]);
 
-  const message = {
-    FAILED: mergingText.failed(),
-    SUCCESS: mergingText.success(),
-    PENDING: mergingText.pending(),
-    MERGING: mergingText.merging(),
-  }[state.status];
+  const loading = React.useContext(LoadingContext);
 
   return (
     <Dialog
       buttons={
-        <Button.Danger onClick={handleClose}>{wbText.stop()}</Button.Danger>
+        <Button.Danger
+          onClick={(): void =>
+            loading(
+              ping(`/api/specify/abort_merging/${mergingId}/`)
+                .then(handleClose)
```

Comment:
Why (when?) using loading + ping over Ajax?

Reply:
and because `ping` doesn't care about the response, you don't have to specify `headers: {Accept ...}`
so it's just used to write less code in this common case and to signal extra clear to those reading code that response does not matter (unless it's an error, in which case it will take care of it on it's own)

ping() does return one thing though (`number`) - the HTTP response code, just in case you need that